### ISSUE TITLE
Is  "visualize_cross_mapping" designed for E=2 specifically?

### ISSUE THREAD

**Issue by lupupu** [PrinceJavier/causal_ccm#2]
https://github.com/PrinceJavier/causal_ccm/blob/1eacde8eff619c8eadabcb756c11a2631eede5c3/causal_ccm/causal_ccm.py#L152

When E=3, the figure should be three-dimensional.

**Comment by PrinceJavier** [comment]
Yes you are right. I only have an implementation for 2D even if E=3. so it only shows t vs t-1 Thinking might it make sense to have an argument to indicate if we want to show 2D for each combination e.g. t vs t-1, t vs t-2, t-1 vs t-2, ... as well as a 3D showing t vs t-1 vs t-2?

**Comment by nsankar** [comment]
@PrinceJavier  There is a new trajectory tracking/plotting package available. perhaps this might be helpful.. https://yupi.readthedocs.io/en/latest/GS/Installation.html
https://yupi.readthedocs.io/en/latest/examples/examples.html
